.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 0.0.1
```

Comment:
This needs to get bumped after Issue #27 gives a tag for the pre-library version of pandamonium.